### PR TITLE
[Gor 25] Create Flutter App

### PR DESCRIPTION
**Pull request by sarahannali**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# gorilla_eats
+
+A new Flutter project.
```
Comment (kevo1ution) [inline README.md]: A better description might be good here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Getting Started
```
Comment (kevo1ution) [inline README.md]: You can get rid of all of this, it is unnecessary (The getting started section)

File: analysis_options.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+include: package:pedantic/analysis_options.1.9.0.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+
+linter:
```
Comment (kevo1ution) [inline analysis_options.yaml:8]: looks good 👍 🥇

File: android/app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
```
Comment (kevo1ution) [inline android/app/build.gradle:40]: Is this something that needs to change? and what is the unique application ID

File: android/app/build.gradle [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        minSdkVersion 16
+        targetSdkVersion 28
+        versionCode flutterVersionCode.toInteger()
+        versionName flutterVersionName
+    }
+
+    buildTypes {
+        release {
+            // TODO: Add your own signing config for the release build.
+            // Signing with the debug keys for now, so `flutter run --release` works.
```
Comment (kevo1ution) [inline android/app/build.gradle:51]: when do we need our own signing config ?

File: android/app/build.gradle (hunk 4 above)
Comment (sarahannali) [reply android/app/build.gradle:40]: its how your app is identified in the google play store, i don't think we need to do it yet

File: android/app/build.gradle (hunk 5 above)
Comment (sarahannali) [reply android/app/build.gradle:51]: according to [this](https://flutter.dev/docs/deployment/android#signing-the-app) we do it when we are about to publish to the google play store

File: README.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# gorilla_eats
```
Comment (kevo1ution) [inline README.md]: This is not python, no need to snake case in readme

**Review by kevo1ution — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (kevo1ution) [reply README.md]: maybe "Gorilla Eats" instead?

**Pull request merged**